modules/appeals_api/config/schemas/shared/v1/address.json

```diff
@@ -22,7 +21,7 @@
             "WI", "WY"
           ]
         },
-        "countryCodeISO2": { "type": "string", "pattern": "^[A-Z]{2}$", "minLength": 2, "maxLength": 2 },
```

Comment:
We actually want to keep these minLength & maxLength keys around - the API Styleguide specifies their inclusion for any string attribute type.

Reply:
that's actually missing in the current files, too ([ref](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/config/schemas/v2/200996.json#L80)), so fixed that in my commit, as well